README.md

```diff
@@ -92,9 +99,8 @@ A community project to get an overview about the Galaxy project
 - [National Microbiology Laboratory's repo](https://github.com/phac-nml/galaxy_tools)
 - [Computational Chemistry](https://github.com/galaxycomputationalchemistry)
 - [Workflow4Metabolomics repo**s**](https://github.com/workflow4metabolomics)
-- [List of usegalaxy.* tools](https://github.com/usegalaxy-eu/usegalaxy-eu-tools)
 - [Fiona Brinkman Lab repo](https://github.com/brinkmanlab/galaxy-tools)
-
+- [Inspect tool](https://github.com/brinkmanlab/galaxy-tools/blob/master/inspect/inspect.xml) reveals resources available to Galaxy tool wrapper command templates
```

Comment:
I'd keep this in the "Tools" section, I think.

Reply:
maybe we should drop it altogether, the main repo is listed above

Reply:
@bgruening I agree, it makes me uneasy to promote this tool.